src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect, useRef, useState } from 'react';
+
+interface Props {
+  title: string;
+  handleChangeTitle: (newTitle: string) => void;
+  exitEditMode: () => void;
+}
+
+export const TodoForm: React.FC<Props> = ({
+  title,
+  handleChangeTitle,
+  exitEditMode,
+}) => {
+  const [newTitle, setNewTitle] = useState(title);
+  const inputReference = useRef<HTMLInputElement>(null);
+  const handleTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
The two names 'handleChangeTitle' and 'handleTitleChange' are a little bit confusing